### PR TITLE
Erxes Add to the Swaglist

### PR DESCRIPTION
**Pull request by gagandeepp**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](../CONTRIBUTING.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

**Comment by gagandeepp** [comment]
@crweiner  Hey buddy ,can you give this a quick look.

File: README.md [hunk 1]
```diff
@@ -171,6 +171,13 @@ For more information regarding this click [here](https://hacktoberfest.digitaloc
 - **How to sign up**: [Grab an issue](https://github.com/earthly/earthly/issues?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest) and after completion we they reach out to you to get your mailing address or reach out to adam@earthly.dev via email to claim your sticker.  If you aren't sure what to work on, [Let's chat](https://gitter.im/earthly-room/community)
 - **Notes**: If you have any questions or need help with anything, join the [Community Gitter Channel]( https://gitter.im/earthly-room/community). They'll gladly help you. See the [blog post](https://blog.earthly.dev/hacktoberfest-2020/) for more details.
 
+#### **Erxes**
+
+- **Swag** : TShirts
+- **Requirements**: Contribute code to the [Erxes](https://github.com/erxes/erxes) open source marketing software.
+- **How to sign up**: [Grab an issue](https://github.com/erxes/erxes/issues) fork the repo ,fix the issue and raise the Pull Request. Once pull request is merged fill out the form at [Swag Request Form]https://erxes.io/hubspot-alternative-erxes-swag
```
[the remark this answers is not in the document]
Comment (crweiner) [reply README.md:178]: Hey @gagandeepp! 👋 Markdown links need to be written like:
```
[Text here](URL Here)
```

I'll take a look at everything else in a few hours.👍

**Comment by crweiner** [comment]
Hello my friend! 👋

Thanks for your contribution to the Hacktoberfest Swag List! Can you please also add the company to the Least Involvement to Most Involvement section and make the quick change to the URL referenced [here](https://github.com/crweiner/hacktoberfest-swag-list/pull/362#discussion_r510331318)

Thanks! 😎

**Comment by gagandeepp** [comment]
Can you please give it a look?

**Comment by gagandeepp** [comment]
Can you merge this @crweiner ?

**Comment by crweiner** [comment]
Hey again! I got all your tags, don't worry. No need to keep posting, my man, I haven't forgotten about you. :smile:

Just wanted to check, does Erxes offer unique swag for Hacktoberfest or is this for all contributions any time of the year? If it's year `round, then Erxes would be just like Kong and not work for this List. [Swag for Dev](https://devswag.io) would be a much better home for swag offered any time of the year. :+1:

**Pull request closed without merging**